modules/vebal/voting-list.prisma

```diff
@@ -0,0 +1,19 @@
+model PrismaVotingList {
+    @@id([address, chain])
+
+    address             String
+    chain               Chain
+
+    isKilled            Boolean
+    createdAt           DateTime    @updatedAt
+    relativeWeightCap   String?
```

Comment:
we already have the status of the gauges (killed, active, preferred) in the gauges table, no need to add it here. Only new field would be the weightCap as far as I understand? For just one field, we might add it to the gauges table instead of creatig a new table here